planner/core/common_plans.go

```diff
@@ -243,31 +248,56 @@ func (e *Execute) OptimizePreparedPlan(ctx context.Context, sctx sessionctx.Cont
 }
 
 func (e *Execute) getPhysicalPlan(ctx context.Context, sctx sessionctx.Context, is infoschema.InfoSchema, prepared *ast.Prepared) error {
+	if prepared.CachedPlan != nil {
+		// expression rewrite will transfer paramMarker in select.where into Constant Expression
+		// but point get execution dose not need to evaluate where condition,
+		// so prepared.UseCache here false is ok, only for point plan
```

Comment:
@yikeke Could you help us to polish this comment?

Reply:
Ok, working on it.

Reply:
Done. @zz-jason My version is the translation of the Chinese explanation given by @cfzjywxk ~